build.sbt

```diff
@@ -252,7 +227,8 @@ lazy val `play-jsonJVM` = `play-json`.jvm.settings(
 
 lazy val `play-json-joda` = project
   .in(file("play-json-joda"))
-  .enablePlugins(PlayLibrary)
+  //  .enablePlugins(PlayLibrary)
+  .enablePlugins(Omnidoc, Publish)
```

Comment:
PlayLibrary was configuring two settings from Omnidoc, see https://github.com/playframework/interplay/blob/master/src/main/scala/interplay/PlayLibraryBase.scala#L24-L25

We need to add here. I think only the first one is relevant though.

Reply:
oh, sorry. I see you have added it later.